utils/utils.go

```diff
@@ -0,0 +1,62 @@
+package utils
+
+import (
+	"fmt"
+	"strings"
+	"time"
+)
+
+func StringToTime(ts string) *time.Time {
+	layout := "2006-01-02 15:04:05.000-07"
```

Comment:
This looks a bit restrictive to only allow a specific format/layout. Could we get more flexibility by letting the user defining the format ?

Like, for example:
```
type Message struct {
  T string `parquet:",timestamp(microsecond,layout=rfc3339)"
}
```

Reply:
I agree about format, just tied up at work at the moment..